src/dashboard/src/components/administration/views_processing.py

```diff
@@ -26,6 +26,7 @@
 from django.http import Http404
 
 from components import helpers
+from processing import install_builtin_config
```

Comment:
How's this going to work when you run Dashboard and MCPServer in separate containers? If you take a look at `dashboard.Dockerfile` you'll notice that the sources of MCPserver are not made available.

If you want to make a function in MCPServer available over the network then you can use our poor Gearman-based RPC interface (soon to become a gRPC-based proper interface). See [RPCserver](https://github.com/artefactual/archivematica/blob/f48521e38ba38a85cec3e07aec9865737d25bbc9/src/MCPServer/lib/RPCServer.py) in https://github.com/artefactual/archivematica/pull/936. Adding new methods to the RPCServer handler is fairly easy.

Reply:
The `processing` module is part of archivematicaCommon so should be present in both containers... unless I've misunderstood something there. Although depending on what's best for the deployment situation, it might be good to have that function available over the network anyway.